systest/mutations-and-queries/mutations_test.go

```diff
@@ -529,13 +622,19 @@ func FacetsOnScalarList(t *testing.T, c *dgo.Dgraph) {
 }
 
 // Shows fix for issue #1918.
-func LangAndSortBugTest(t *testing.T, c *dgo.Dgraph) {
+func (ssuite *SystestTestSuite) LangAndSortBugTest() {
+	t := ssuite.T()
```

Comment:
The next few lines seem common to me in all of the test. Is it possible to move this to somewhere common? Like, we can define a common function in a test file and use it in both integration_test.go and upgrade_test.go in the test setup.